lib/ansible/modules/cloud/amazon/elasticache.py

```diff
@@ -481,31 +481,31 @@ def _get_nodes_to_remove(self):
 
 
 def main():
-    argument_spec = ec2_argument_spec()
+    """ elasticache ansible module """
+    argument_spec             = ec2_argument_spec()
     argument_spec.update(dict(
-        state                 ={'required': True, 'choices': ['present', 'absent', 'rebooted']},
-        name                  ={'required': True},
-        engine                ={'required': False, 'default': 'memcached'},
-        cache_engine_version  ={'required': False},
-        node_type             ={'required': False, 'default': 'cache.m1.small'},
-        num_nodes             ={'required': False, 'default': None, 'type': 'int'},
+        state                 = {'required': True, 'choices': ['present', 'absent', 'rebooted']},
```

Comment:
Yeah, it is for me too. But then when we add another options with a long name we'll have a gigantic diff to readjust everything. Easiest just to remove the spaces once and then have one line diffs :)